src/Uno.UI/UI/Xaml/Window.WinAppSdk.cs

```diff
@@ -0,0 +1,21 @@
+﻿#nullable enable
+#if HAS_UNO_WINUI
+
+using Windows.UI.ViewManagement;
+
+namespace Windows.UI.Xaml;
+
+public sealed partial class Window
+{
+#if !__IOS__ // This can be added when iOS uses SceneDelegate #8341.
+	/// <summary>
+	/// Gets or sets a string used for the window title.
+	/// </summary>
+	public string Title
+	{
+		get => ApplicationView.GetForCurrentView().Title;
+		set => ApplicationView.GetForCurrentView().Title = value;
+	}
+#endif
+}
+#endif
```

Comment:
Why not `#if HAS_UNO_WINUI && !__IOS__` together at the beginning?

Reply:
There will be more "WinAppSDK-specific" code for `Window`, so to make future changes easier, the file itself is Uno.WinUI, but just the property is not supported on iOS yet.